src/Event.cc

```diff
@@ -160,7 +152,7 @@ void EventMgr::Dispatch(Event* event, bool no_remote)
 void EventMgr::Drain()
 	{
 	if ( event_queue_flush_point )
-		Enqueue(event_queue_flush_point, Args{});
+		Enqueue(event_queue_flush_point, Args {});
```

Comment:
This one has happened a few times earlier but I'm now bothered enough by it to comment. This is probably a bug in clang-format in that it doesn't know that it's an initialization and not a new block.

Reply:
Does SpaceBeforeCpp11BracedList control this?

Reply:
Yep, that was it. I had it set to `true` in the config.